### PR TITLE
generic json rich ui

### PR DESCRIPTION
**Pull request by jgrayson676**
## Description of the change

adds an RUI for the json integration

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change
- [ ] Technical Debt
- [ ] Documentation

## Related tickets

https://app.clubhouse.io/active-prospect/story/11332/update-generic-json-rui

### Development and Testing

- [ ]  Lint rules pass locally.
- [ ]  The code changed/added as part of this pull request has been covered with tests, or this PR does not alter production code.
- [ ]  All tests related to the changed code pass in development, or tests are not applicable.

### Code Review

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached.
- [ ]  At least two engineers have been added as "Reviewers" on the pull request.
- [ ]  Changes have been reviewed by at least two other engineers who did not write the code.
- [ ]  This branch has been rebased off master to be current.

### Tracking 
- [ ]  Issue from Clubhouse has a link to this pull request.
- [ ]  This PR has a link to the issue in Clubhouse.

### QA
- [ ]  This branch has been deployed to staging and tested.

### CODE REVIEW

**Review by cgrayson — CHANGES_REQUESTED** [review]
Some UX changes to make on this, as discussed offline. 🙇

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "flat": "^5.0.2",
     "leadconduit-custom": "^2.16.0",
     "leadconduit-integration-ui": "^1.1.4",
+    "leadconduit-sanitize-name": "^1.0.2",
```
Comment (jgrayson676) [inline package.json:28]: I wonder if it's worth adding this to the `dev-dependencies` package, any recipient integration with a rui uses it.

Comment (cgrayson) [reply package.json:28]: Good idea 😁 Now that we added it, you can take this out.

File: package.json [hunk 2]
```diff
@@ -16,10 +20,18 @@
...
+    "leadconduit-integration-ui": "^1.1.4",
+    "leadconduit-sanitize-name": "^1.0.2",
+    "vue": "^2.6.12",
+    "vue-router": "^3.5.1",
+    "vuex": "^3.6.2"
   },
   "devDependencies": {
-    "chai": "^4.1.2",
-    "mocha": "^8.3.0"
+    "@activeprospect/integration-components": "^2.0.0",
```
Comment (cgrayson) [inline package.json:34]: I still wonder about if we could get away with everything that ends up webpack'd as a `devDependency`, but when I've used `integration-components` I've made it a production `dependency`.

**Review by cgrayson — APPROVED** [review]
Otherwise LGTM

File: package.json (hunk 2 above)
Comment (jgrayson676) [reply package.json:34]: I don't mind changing it, but it seems to have worked fine in staging at least

**Review by cgrayson — APPROVED** [review]
LGTM

**Pull request merged**